src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction() {
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(String code) {

```

Comment:
The method setOperation is taking a String parameter while it should take an Operation as parameter to set the operation.

Reply:
ok, done